app/views/admin/developers/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+  <div class="col s12 center">
+  	<h2 class="header gray-text2">Membro</h2>
+  </div>
+</div>
+<div class="col s12 m8 offset-m2 l6 offset-l3">
+	<div class="card-panel grey lighten-5 z-depth-1">
+	  <div class="row valign-wrapper">
+	    <div class="col s2">
+	      <%= image_tag attachment_url(@developer, :profile_image, :fill, 300, 300, format: "jpg", fallback: "default-avatar"), :class => 'circle responsive-img profile-image' %>
+	    </div>
+	    <div class="col s10">
+	      <span class="black-text"> 
+					<h1>Name: <%= @developer.name %> </h1>
+					<h2>Função: <%= @developer.function %></h2>
+					<h2>Instituição: <%= @developer.institution %></h2>
+					<% if @developer.active == true %>
```

Comment:
This can be done with a helper method.
```
  def humanize_boolean(boolean, {_true: nil, _false: })
    I18n.t((!!boolean).to_s)
  end
```
So in the view just use

```
<td><p><%= humanize_boolean(deve.active?) %></p></td>
```

config/locales/pt-BR.yml
```
'pt-BR':
  :true: 'Sim'
  :false: 'Não'
```